### PR TITLE
Inform the user only once for each loaded script engine

### PR DESCRIPTION
**Pull request by triller-telekom**
Relates to #3589

Signed-off-by: Stefan Triller <stefan.triller@telekom.de>

### CODE REVIEW

File: bundles/automation/org.eclipse.smarthome.automation.module.script/src/main/java/org/eclipse/smarthome/automation/module/script/internal/GenericScriptEngineFactory.java [hunk 1]
```diff
@@ -16,9 +16,19 @@
...
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 public class GenericScriptEngineFactory implements ScriptEngineFactory {
     private ScriptEngineManager engineManager = new ScriptEngineManager();
+    private final Logger logger = LoggerFactory.getLogger(getClass());
+
+    public GenericScriptEngineFactory() {
+        for (javax.script.ScriptEngineFactory f : engineManager.getEngineFactories()) {
+            logger.info("Supported script engine: {} for language {} with mimetypes {}", f.getEngineName(),
```
Comment (kaikreuzer) [inline bundles/automation/org.eclipse.smarthome.automation.module.script/src/main/java/org/eclipse/smarthome/automation/module/script/internal/GenericScriptEngineFactory.java]: As it is INFO level that most end users will get to see in their main log, I would suggest to make it a bit more end user compatible, e.g.

> Activated scripting support for ECMAScript

We could add an additional debug log that then also lists all possible file extensions and mimetypes as well as the engine name itself.

Comment (triller-telekom) [inline bundles/automation/org.eclipse.smarthome.automation.module.script/src/main/java/org/eclipse/smarthome/automation/module/script/internal/GenericScriptEngineFactory.java]: Done.

File: bundles/automation/org.eclipse.smarthome.automation.module.script/src/main/java/org/eclipse/smarthome/automation/module/script/internal/GenericScriptEngineFactory.java [hunk 2]
```diff
@@ -16,9 +16,22 @@
...
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 public class GenericScriptEngineFactory implements ScriptEngineFactory {
     private ScriptEngineManager engineManager = new ScriptEngineManager();
+    private final Logger logger = LoggerFactory.getLogger(getClass());
+
+    public GenericScriptEngineFactory() {
+        for (javax.script.ScriptEngineFactory f : engineManager.getEngineFactories()) {
+            logger.info("Activated scripting support for language {}", f.getLanguageName());
```
Comment (kaikreuzer) [inline bundles/automation/org.eclipse.smarthome.automation.module.script/src/main/java/org/eclipse/smarthome/automation/module/script/internal/GenericScriptEngineFactory.java]: could you remove the word "language"? I think this is obsolete here.

Comment (triller-telekom) [inline bundles/automation/org.eclipse.smarthome.automation.module.script/src/main/java/org/eclipse/smarthome/automation/module/script/internal/GenericScriptEngineFactory.java]: Done

**Pull request merged**